be/src/exprs/runtime_filter.h

```diff
@@ -459,6 +471,155 @@ struct WithModuloArg {
             }
         }
     };
+
+    template <TRuntimeFilterLayoutMode::type M>
+    struct HashValueComputeWithSelection {
```

Comment:
This code is tedious and error-prone,  seems duplicate, suggest to unite it with HashValueCompute into one.

Reply:
fixed by adding a template parameter to represent iterator